frontend/src/components/common/HeaderBar.js

```diff
@@ -0,0 +1,50 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import { Layout, Menu, Input } from "antd";
+const { Header: AntHdr } = Layout;
+
+const Header = props => (
+    <AntHdr>
+        <Menu
+            theme='dark'
+            mode='horizontal'
+            defaultSelectedKeys={["signup"]}
+            selectable={false}
+            style={{ lineHeight: "64px" }}
+        >
+            <Menu.Item key='home'>
+                <Link to='/'>
+                    <img
+                        height='48'
+                        width='48'
+                        alt='logo'
+                        src={require("../../assets/GaleriLogo.png")}
+                        style={{ backgroundColor: "#ffffff", borderRadius: "10px", margin: "5px" }}
+                    />
+                    <span style={{ fontWeight: "bold", fontSize: "1.25em" }}>
+                        {props.title}
+                    </span>
+                </Link>
+            </Menu.Item>
+            {/* <Menu.Item key='about'>
+                <Link to='/about'>About</Link>
+            </Menu.Item> */}
+
+            <Menu.Item key='search'>
+                <Input.Search
+                    placeholder='Search for projects'
+                    onSearch={query => props.searchFunc(query)}
```

Comment:
Should probably follow React conventions: `onSearch` or `onSearchHandle`? Could you look it up in the React tic-tac-toe tutorial?

Reply:
It'd be handleSearch, but onSearch is in the [API](https://ant.design/components/input/#Input.Search)